diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5f91a33..e6ab784 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -85,5 +85,6 @@ catkin_package(
 )
 
 add_executable(fastlio_mapping src/laserMapping.cpp include/ikd-Tree/ikd_Tree.cpp src/preprocess.cpp)
+add_dependencies(fastlio_mapping ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
 target_link_libraries(fastlio_mapping ${catkin_LIBRARIES} ${PCL_LIBRARIES} ${PYTHON_LIBRARIES})
-target_include_directories(fastlio_mapping PRIVATE ${PYTHON_INCLUDE_DIRS})
\ No newline at end of file
+target_include_directories(fastlio_mapping PRIVATE ${PYTHON_INCLUDE_DIRS})
